### PR TITLE
Several small fixes

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
See commits

### CODE REVIEW

File: frontend/src/layout/header/Search.tsx [hunk 1]
```diff
@@ -19,13 +19,20 @@ export const SearchField: React.FC<SearchFieldProps> = ({ variant }) => {
...
-            if (document.activeElement?.tagName === "INPUT") {
+
+            // With ctrl and meta key, this could be bound to some other
+            // shortcut (ctrl+s) that we want to ignore.
+            if (ev.ctrlKey || ev.metaKey) {
                 return;
             }
-            if (ev.key === "s" || ev.key === "S") {
+
+            console.log(ev.key);
```
Comment (JulianKniephoff) [inline frontend/src/layout/header/Search.tsx:34]:
```suggestion
```

Whoopsydoopsy?

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Just a small thing.

File: frontend/src/layout/header/Search.tsx (hunk 1 above)
Comment (LukasKalbertodt) [reply frontend/src/layout/header/Search.tsx]: Ooopsie daisy!

**Review by JulianKniephoff — APPROVED** [review]
:ok_hand:

**Pull request merged**